lib/tasks/dev.rake

```diff
@@ -0,0 +1,63 @@
+require 'rspec/core'
+require 'rspec/core/rake_task'
+require 'solr_wrapper'
+require 'fcrepo_wrapper'
+require 'active_fedora/rake_support'
+
+module DevRakeUtil
+  def load_config(service, environment, defaults)
```

Comment:
Maybe comment what this is overriding? Something in active_fedora or one / both of the wrappers?